.github/workflows/Linter_Check.yml

```diff
@@ -0,0 +1,20 @@
+
+ 
+
+name: Super-Linter
+
+on: push
+
+jobs:
+  super-lint:
+    name: Lint code base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Super-Linter
+        uses: github/super-linter@v4
+        env:
```

Comment:
Since we utilize `ESLint` for JavaScript files (and that it's failing with `standard js`), you should probably disable using `standard js` by adding:
```
VALIDATE_ALL_CODEBASE: true
VALIDATE_JAVASCRIPT_STANDARD: false
```
- Ref: https://www.freecodecamp.org/news/github-super-linter#howtoaddspecificlinterstosuperlinter